src/organisation/containers/organisation/organisation.component.spec.ts

```diff
@@ -174,4 +189,61 @@ describe('OrganisationComponent', () => {
     expect(store.pipe).toHaveBeenCalled();
     expect(component.getOrganisationDetails()).toEqual(mockOrganisationDetails);
   });
+
+  it('should get the User Details from the Store, and set it on orgData.', () => {
+
+    component.getOrganisationDetailsFromStore();
+
+    expect(store.pipe).toHaveBeenCalled();
+    expect(component.getOrganisationDetails()).toEqual(mockOrganisationDetails);
+  });
+
+  describe('showChangePbaNumberLink property', () => {
+    const CHANGE_LINK = By.css('#change-pba-account-numbers__link');
+
+    it('should show the \'Change\' link when false', () => {
+      component.showChangePbaNumberLink = true;
+      fixture.detectChanges();
+
+      const changeLink = fixture.debugElement.query(CHANGE_LINK);
+
+      expect(changeLink).toBeTruthy();
+    });
+
+    it('should not show the \'Change\' link when false', () => {
+      component.showChangePbaNumberLink = false;
+      fixture.detectChanges();
+
+      const changeLink = fixture.debugElement.query(CHANGE_LINK);
+
+      expect(changeLink).toBeNull();
+    });
+  });
+
+  describe('canShowChangePbaNumbersLink()', () => {
+
+    it('should set to true when the user has pui-finance-manager and there are PBA accounts', () => {
+      userIsPuiFinanceManager = true;
+      component.organisationPaymentAccount = [
+        'PBA00000'
+      ];
+
+      component.canShowChangePbaNumbersLink();
+
+      fixture.detectChanges();
```

Comment:
Does this need change detection? What would happen if this line wasn't here?

Reply:
done

Reply:
You were right, it didn't need this so I removed.